packages/web-components/tests/e2e-storybook/cypress/integration/footer/footer-short.e2e.js

```diff
@@ -80,18 +83,17 @@ describe('c4d-footer | Short (desktop)', () => {
     cy.takeSnapshots();
   });
 
-  it('should display interactive search field and with keywords for locations and languages', () => {
+  it.skip('should display interactive search field and with keywords for locations and languages', () => {
```

Comment:
Missed `skip`? or is this deliberate?

Reply:
All the `.skip`'s here are intentional. My approach here was not to go through the entire suite and debug and fix each one, but to cover the most obvious fixes (prefix updates), review the result, and for anything more complex needing a deeper understanding of the individual test, mark as skipped for later follow up.

The idea is, even with all the tests marked as skipped, there is still plenty of value here that could be useful while we chip away at the skipped tests.

Reply:
That works for me. After we finish the v2 upgrade, I can split the remaining `skips` into tickets for the team and we can look at what holes we might have in coverage.